core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -125,8 +125,17 @@ class PluginTypeChecker(val platform: Platform = PlatformFactory.createInstance(
       case None =>
         logInfo(s"Reading operators scores with platform: $platform")
         val file = platform.getOperatorScoreFile
-        val source = Source.fromResource(file)
-        readOperators(source, "score", true).map(x => (x._1, x._2.toDouble))
+        try {
+          val source = Source.fromResource(file)
+          readOperators(source, "score", true).map(x => (x._1, x._2.toDouble))
+        } catch {
+          case NonFatal(_) =>
+            logWarning(s"Unable to read operator scores from file: $file")
+            val defaultFile = platform.getDefaultOperatorScoreFile
+            logInfo(s"Using default operator scores file: $defaultFile")
```

Comment:
I would make both log statements warnings and put them together in one line

Reply:
Thanks @tgravescs! Addressed all feedback in new iteration.